h/static/scripts/controllers/share-widget-controller.js

```diff
@@ -0,0 +1,211 @@
+'use strict';
+
+const Controller = require('../base/controller');
+
+const TEMPLATE_SELECTOR = '.share-widget-template';
```

Comment:
In our CSS style guide, we specify that all classes referenced by code should use a `js-` prefix, except in tests: https://github.com/hypothesis/frontend-toolkit/blob/master/docs/css-style-guide.md#classes-used-in-code